config/sets/http-solid.json

```diff
@@ -0,0 +1,29 @@
+{
+  "@context": [
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-init-sparql/^1.0.0/components/context.jsonld",
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/runner/^1.0.0/components/context.jsonld",
+
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-http-solid-auth-fetch/^1.0.0/components/context.jsonld",
+
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/bus-http/^1.0.0/components/context.jsonld",
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/mediator-number/^1.0.0/components/context.jsonld"
+  ],
+  "@graph": [
+    {
+      "@id": "urn:comunica:my",
+      "actors": [
+        {
+          "@id": "config-sets:http.json#myHttpFetcher",
+          "@type": "ActorHttpSolidAuthFetch"
+        }
+      ]
+    },
+    {
+      "@id": "config-sets:http.json#mediatorHttp",
+      "@type": "MediatorNumberMin",
+      "field": "time",
+      "ignoreErrors": true,
```

Comment:
meaning?

Reply:
This is the same as our default HTTP mediator. This means that if multiple actors are in the bus, and one of them rejects a test, the next one will be checked. Only if all actors reject, then the mediator will reject as well.